server/timestamp/timestamp.go

```diff
@@ -95,16 +93,15 @@ func timestampExpired(ts *data.SignedTimestamp) bool {
 	return signed.IsExpired(ts.Signed.Expires)
 }
 
+// snapshotExpired verify the checksum(s) for the given snapshot via timestamp's data field.
```

Comment:
nit: "snapshotExpired verifies the checksum(s) for the given snapshot using metadata from the timestamp"